nautobot/extras/filters/__init__.py

```diff
@@ -297,6 +290,13 @@ class ConfigContextFilterSet(BaseFilterSet):
         to_field_name="slug",
         label="Tag (slug)",
     )
+    # TODO(timizuo): Limit Role choices by Device/VM but can't
+    #  import device cause of partial import error
+    roles = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
Should probably be `role` (as in the removed code above) as the other filters are single rather than plural as well.